config/e2e/kustomization.yaml

```diff
@@ -0,0 +1,5 @@
+namespace: kubestone-system
```

Comment:
just wondering: might be a good idea to instantiate into a dedicated e2e namespace, so that it won't conflict with real installations. WDYT?